doc/src/manual/noteworthy-differences.md

```diff
@@ -127,8 +127,9 @@ For users coming to Julia from R, these are some noteworthy differences:
     matrices, then `A * B` denotes a matrix multiplication in Julia, equivalent to R's `A %*% B`.
     In R, this same notation would perform an element-wise (Hadamard) product. To get the element-wise
     multiplication operation, you need to write `A .* B` in Julia.
-  * Julia performs matrix transposition using the `.'` operator and conjugated transposition using
-    the `'` operator. Julia's `A.'` is therefore equivalent to R's `t(A)`.
+  * Julia performs matrix transposition using the `transpose` function and conjugated transposition using
+    the `'` operator or `adjoint` function. Julia's `transpose(A)` is therefore equivalent to R's `t(A)`.
+    Additionally a non-recursive transpose in Julia is provided by `permutedims` function.
```

Comment:
or *the* \`adjoint\` function

by the \`permutedims\` function

Reply:
fixed